src/job/mod.rs

```diff
@@ -106,7 +106,7 @@ pub async fn init() -> Result<(), anyhow::Error> {
 
     // Auth auditing should be done by router also
     #[cfg(feature = "enterprise")]
-    if self_reporting::run_audit_publish().await.is_none() {
+    if self_reporting::run_audit_publish().is_none() {
```

Comment:
This is `async` function

Reply:
the function was modified to be a sync function in this pr
https://github.com/openobserve/openobserve/pull/7996